.github/workflows/aptos-framework-test.yaml

```diff
@@ -36,5 +36,5 @@ jobs:
  
     - name: Run tests
       run: |
-        ${{ env.CARGO_BIN }} test --release -p aptos-framework -- --skip prover
+        ${{ env.CARGO_BIN }} test --release -p ./aptos-move/framework/supra-framework -- --skip prover
```

Comment:
Hi @sjadiya-supra , this line doesn't have any problem and should not be changed. In fact, there is no problem with `yaml` file. You need to ensure that runner supports `cargo nextest` command. (See line 40) on this file. So in CI jobs, the tests on line 39 goes through but line 40 does not because it can not find 'nextest'.  It's a package/crate that you need to install on the runner See: [https://nexte.st/](https://nexte.st/)